sgtm/artemis/README.md

```diff
@@ -196,22 +196,23 @@ This service account needs to have permission to access the Firestore data.
 5. Select the "[Artemis variable template](./get_document_from_firestore.tpl)”
     file.
 6. Go to the permission tab and set the permissions for Firestore, ensuring you
-   update the project ID.
-   ![Template permissions](./img/gtm-template-permissions.png)
-7. Save the template.
-8. Go to variables -> new user defined variable and create a “user” variable
+   update the project ID and the collection name. Use a * to allow the tag to 
+   access any collection. Incorrect details here will produce a permission
+   error when you fire the tag.
+8. Save the template.
```

Comment:
nit: wrong numbering. Although I think the MD is rendered correctly.

It's also making the change bigger than it actually is.